PubNub/Network/Parsers/PNSubscribeParser.m

```diff
@@ -225,14 +221,14 @@ + (NSMutableDictionary *)eventFromData:(id)data forChannel:(NSString *)channel
         [event addEntriesFromDictionary:[self messageFromData:data
                                      withAdditionalParserData:additionalData]];
     }
-    
+//
     return event;
 }
 
 + (NSMutableDictionary *)messageFromData:(id)data
                 withAdditionalParserData:(NSDictionary *)additionalData {
     
-    NSMutableDictionary *message = [@{@"message":data} mutableCopy];
+    NSMutableDictionary *message = [@{@"message":data[kPNMessagePayloadDataKey]} mutableCopy];
```

Comment:
@jzucker2 Because _data[kPNMessagePayloadDataKey]_ will be passed from calling method, there is no need in this and this line can be reverted to original one.